gradle-baseline-java/src/main/java/com/palantir/baseline/plugins/BaselineErrorProne.java

```diff
@@ -63,6 +64,11 @@ private static void applyToJavaProject(Project project) {
                         new ConditionalPatchCheck(
                                 new IfModuleIsUsed("com.palantir.safe-logging", "logger"), "PreferSafeLogger"));
 
+        project.getTasks().withType(JavaCompile.class).configureEach(javaCompile -> {
+            // This is superseded by the DeprecatedForRemovalApiUsage error-prone check
+            javaCompile.getOptions().getCompilerArgs().add("-Xlint:-removal");
```

Comment:
You should use a `CommandLineArgumentProvider` here and not just addd to the `getCompilerArgs()` - the latter will force the evaluation of the existing command line args, which could break things.